app/models/project_step.rb

```diff
@@ -197,6 +212,29 @@ def week_of_scheduled_date
   end
 
   #
+  # batchable actions
+  #
```

Comment:
Please remove this comment block. I don't think it's necessary and big block comments like this are a code smell in my opinion, since they suggest that there is some cohesive set of behavior that should be moved into a composition or mixin arrangement.